src/config/wallets.ts

```diff
@@ -188,3 +188,10 @@ export const supportAllWalletsObj = {
 export const supportEvmWallets = objToArray(supportEvmWalletObj) as Wallet[];
 export const supportWallets = objToArray(supportWalletObj) as Wallet[];
 export const supportAllWallets = objToArray(supportAllWalletsObj) as Wallet[];
+
+// Todo: get the faucet address from faucet-bot-api
+export const faucetHotwallet = 'WiXeZpx9gX9T3TRcbYanyDCk64zqaCoo1EQvssnTiywWgQB';
```

Comment:
Working on getting address from the API
https://github.com/AstarNetwork/astar-faucet-bot/pull/47